include/odp/api/spec/schedule.h

```diff
@@ -45,6 +46,53 @@ extern "C" {
  * Maximum schedule group name length in chars including null char
  */
 
+/**
+ * Schedule configuration options
+ */
+typedef struct odp_schedule_config_t {
+
+	/* Scheduler flow aware
+	 * Application can configure the scheduler to be flow aware or unaware.
+	 * When the scheduler is configured as flow aware the ORDERED and
+	 * ATOMIC schedule queue synchronization by the scheduler is performed
+	 * per flow within each queue.
+	 */
+	odp_support_t flow_aware;
```

Comment:
An `odp_support_t` makes sense in a capability but a configuration sets a specific mode and hence should be a boolean.

Reply:
Is this option really needed, since setting `odp_schedule_config_t .flow_count` to 1 means practically the same thing?

Reply:
Agree, single parameter (flow_count) can be used to enable the feature.